### PR TITLE
Desc automatic update

### PR DESCRIPTION
**Pull request by l0k0ms**
This will allow to inline the short description in the manifest.json file of each integration as the link preview

to call the description from the manifest just use: {{< get-desc-from-git >}}

(sorry for the amount of commits.. I tried to rebase but It messed up.. will try again next time)

There is some tests inside this code to avoid dogweb integrations, this will be removed once the logic will be iso between integration core and dogweb.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -39,6 +39,7 @@ build_preview:
     - post_dd_event "documentation deploy ${CI_COMMIT_REF_NAME} started" "${CI_PROJECT_URL}/pipelines/${CI_PIPELINE_ID}" "info"
     - version_static_assets
     - placehold_translations
+    - update_pre_build
```
Comment (mstbbs) [inline .gitlab-ci.yml]: let's name this something more explicit, perhaps `- sync_integration_descriptions`?

File: local/bin/sh/helpers.sh [hunk 2]
```diff
@@ -155,6 +155,9 @@ sync_integration_metrics() {
 	integrations_sync.py --token ${GITHUB_TOKEN}
 }
 
+update_pre_build() {
+    update_pre_build.py --token ${GITHUB_TOKEN}
+}
```
Comment (mstbbs) [inline local/bin/sh/helpers.sh]: let's decorate this with the `start_step`, `pass_step`, and `|| fail_step` functions. These will standardize the bash calls and report to datadog. see _most_ of the other helpers as examples.

File: .gitlab-ci.yml (hunk 1 above)
Comment (l0k0ms) [inline .gitlab-ci.yml]: 👍

File: local/bin/sh/helpers.sh (hunk 2 above)
Comment (l0k0ms) [inline local/bin/sh/helpers.sh]: 👍

**Comment by l0k0ms** [comment]
This is the first step before the readme scrapper. logic might change soon

**Pull request merged**